### ISSUE TITLE
Autoplay video function appears to no longer work with 0.8.1 quickfix

### ISSUE THREAD

**Issue by keLston** [ParticleCore/Particle#144]
Browser Name: Firefox

Browser Version: 42.0

Operating System: Windows 7 x64

Software Type: Addon

Software Version: 0.8.1

Userscript Manager: n/a

Userscript Version: n/a

Problem Description: With autoplay video option selected, videos no longer autoplay.

Steps to reproduce: Updated to the 0.8.1 for the Youtube Subscription Grid quickfix, change no other settings, attempt to load a video.

Additional details: I tried unchecking autoplay video -> saving the setting -> reload video -> then reselecting autoplay videos to see if I could retoggle the setting but did not appear to have an effect

**Comment by ParticleCore** [comment]
I am unable to replicate your problem, all videos are playing automatically when I disable autoplay. Have you disabled all other add-ons and userscripts except YT+ to see if the problem persists?

**Comment by q1k** [comment]
There are two autoplay options.

One is from YouTube - playing next video automatically.

The other is from your script - which auto starts the video on page load. It's misleading as it says autoplay videos.
It should say auto start playing videos on page load.

**Comment by keLston** [comment]
I figured out the problem just now after reloading the browser a number of times.

It turned out that Firefox decided the best way to deal with installing the .xpi off Github was install a second copy of Youtube+ so I had 0.5.8 and 0.8.1 installed simultaneously which, for some reason, did not show up on the Extensions page the first time I looked. This was the source of the problem as removing 0.5.8 fixed the issue.
